embed_darwin_arm64.go

```diff
@@ -9,7 +9,7 @@ package waf
 
 import _ "embed" // Needed for go:embed
 
-//go:embed lib/darwin-arm64/_libddwaf.dylib
+//go:embed lib/darwin-arm64/libddwaf.dylib
```

Comment:
I can't recall exactly but before all the files (even on linux) were named this way because there was a conflit with the static libraries. But I kept the ones of OSX because the linker was doing garbage-tier errors, like interpreting this line as a dependency to load which was reaaaaally wierd (as always). We can try to check if this is happening again